### PR TITLE
Add the default rules validation in the switch validation.

### PR DESCRIPTION
**Pull request by dpoltavets**

### CODE REVIEW

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/rule/FlowEntry.java [hunk 1]
```diff
@@ -19,13 +19,16 @@
...
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 
 import java.io.Serializable;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Value
 @Builder
+@EqualsAndHashCode(exclude = {"durationSeconds", "durationNanoSeconds", "packetCount", "byteCount", "tableId",
+        "idleTimeout", "hardTimeout", "version"})
```
Comment (niksv) [inline services/src/messaging/src/main/java/org/openkilda/messaging/info/rule/FlowEntry.java]: I'm wondering about 'tableId'.

Comment (dpoltavets) [reply services/src/messaging/src/main/java/org/openkilda/messaging/info/rule/FlowEntry.java]: 'tableId' has been removed.

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java [hunk 2]
```diff
@@ -89,11 +93,47 @@ private ValidateRulesResult makeRulesResponse(Set<Long> expectedCookies,
                     cookiesIntoLogRepresentation(excessRules));
         }
 
+        List<FlowEntry> presentDefaultRules = presentRules.stream()
```
Comment (timofei-durakov) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: please extract this to private metrod

Comment (dpoltavets) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: Done.

**Pull request merged**